backend/models/discord_user.py

```diff
@@ -1,14 +1,16 @@
 import typing as t
 
-from pydantic import BaseModel
+from pydantic import BaseModel, validator
 
 
 class DiscordUser(BaseModel):
     """Schema model of Discord user for form response."""
 
-    # Discord default fields
-    id: int  # This is actually snowflake, but we simplify it here
+    # Discord default fields.
     username: str
+    # Store ID as str not as int because JavaScript
+    # doesn't work correctly with big ints.
```

Comment:
We don't need this comment, the data is already a string when it comes from Discord.
```suggestion
```